Index: openvswitch-2.3.90/Makefile.am
===================================================================
--- openvswitch-2.3.90.orig/Makefile.am	2016-10-25 15:29:13.193473506 +0200
+++ openvswitch-2.3.90/Makefile.am	2016-10-25 15:29:29.125473181 +0200
@@ -7,7 +7,6 @@
 
 AUTOMAKE_OPTIONS = foreign subdir-objects
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = datapath
 
 AM_CPPFLAGS = $(SSL_CFLAGS)
 AM_LDFLAGS = $(SSL_LDFLAGS)
@@ -343,11 +342,6 @@
 uninstall-local: $(UNINSTALL_LOCAL)
 .PHONY: $(DIST_HOOKS) $(CLEAN_LOCAL) $(INSTALL_DATA_LOCAL) $(UNINSTALL_LOCAL)
 
-modules_install:
-if LINUX_ENABLED
-	cd datapath/linux && $(MAKE) modules_install
-endif
-
 dist-docs:
 	VERSION=$(VERSION) $(srcdir)/build-aux/dist-docs $(srcdir) $(docs)
 .PHONY: dist-docs
@@ -360,13 +354,4 @@
 include include/automake.mk
 include third-party/automake.mk
 include debian/automake.mk
-include vswitchd/automake.mk
 include ovsdb/automake.mk
-include rhel/automake.mk
-include xenserver/automake.mk
-include python/automake.mk
-include python/compat/automake.mk
-include tutorial/automake.mk
-include vtep/automake.mk
-include datapath-windows/automake.mk
-include datapath-windows/include/automake.mk
Index: openvswitch-2.3.90/acinclude.m4
===================================================================
--- openvswitch-2.3.90.orig/acinclude.m4	2016-10-25 15:29:13.193473506 +0200
+++ openvswitch-2.3.90/acinclude.m4	2016-10-25 15:29:13.193473506 +0200
@@ -140,7 +140,7 @@
          AC_ERROR([Linux kernel in $KBUILD is version $kversion, but version newer than 3.19.x is not supported (please refer to the FAQ for advice)])
        fi
     else
-       if test "$version" -le 1 || test "$patchlevel" -le 5 || test "$sublevel" -le 31; then
+       if test "$version" -le 1 || test "$patchlevel" -le 5 || test "$sublevel" -le 27; then
          AC_ERROR([Linux kernel in $KBUILD is version $kversion, but version 2.6.32 or later is required])
        else
          : # Linux 2.6.x
